core/src/test/java/com/graphhopper/routing/lm/LandmarkStorageTest.java

```diff
@@ -228,4 +240,66 @@ public BBox getBounds() {
         storage.createLandmarks();
         assertEquals(3, storage.getSubnetworksWithLandmarks());
     }
+
+    @Test
+    public void testDelta() {
+        int distance = 1000000;
+
+        ghStorage.edge(1, 2, distance, false);
+        ghStorage.edge(2, 3, distance, false);
+        ghStorage.edge(3, 1, distance, false);
+
+        ghStorage.edge(2, 4, distance, true);
+        ghStorage.edge(4, 5, distance, true);
+        ghStorage.edge(5, 6, distance, true);
+        ghStorage.edge(6, 7, distance, true);
+        ghStorage.edge(7, 8, distance, true);
+        ghStorage.edge(8, 9, distance, true);
+
+        ghStorage.edge(3, 10, distance, true);
+        ghStorage.edge(10, 11, distance, true);
+        ghStorage.edge(11, 12, distance, true);
+        ghStorage.edge(12, 13, distance, true);
+        ghStorage.edge(13, 14, distance, true);
+        ghStorage.edge(14, 15, distance, true);
+
+        LandmarkStorage storage = new LandmarkStorage(ghStorage, new RAMDirectory(), new FastestWeighting(encoder), 2);
+        storage.createLandmarks(); //landmarks: 15, 9
+
+        assertEquals(35680, storage.getFromWeight(0,1));
+        assertEquals(40777, storage.getToWeight(0,1));
+        assertEquals(71361, storage.getFromWeight(0,9));
+        assertEquals(66264, storage.getToWeight(0,9));
+        assertEquals(15291, storage.getFromWeight(0,12));
+        assertEquals(15291, storage.getToWeight(0,12));
+        assertEquals(40777, storage.getFromWeight(1,1));
+        assertEquals(35680, storage.getToWeight(1,1));
+        assertEquals(50972, storage.getFromWeight(1,12));
+        assertEquals(56069, storage.getToWeight(1,12));
+        assertEquals(66264, storage.getFromWeight(1,15));
+        assertEquals(71361, storage.getToWeight(1,15));
+    }
+
+    @Test
+    public void testDeltaWarning() {
+        int distance = 1000000;
+
+        ghStorage.edge(1, 2, distance, false);
+        ghStorage.edge(2, 3, distance, false);
+        ghStorage.edge(3, 4, distance, false);
+        ghStorage.edge(4, 5, distance, false);
+        ghStorage.edge(5, 6, distance, false);
+        ghStorage.edge(6, 1, distance, false);
+
+        ghStorage.edge(1, 7, distance, true);
+        ghStorage.edge(7, 8, distance, true);
+        ghStorage.edge(8, 9, distance, true);
+
+        ghStorage.edge(6, 10, distance, true);
+        ghStorage.edge(10, 11, distance, true);
+        ghStorage.edge(11, 12, distance, true);
+
+        LandmarkStorage storage = new LandmarkStorage(ghStorage, new RAMDirectory(), new FastestWeighting(encoder), 2);
+        storage.createLandmarks(); //landmarks: 12, 9
```

Comment:
Here we should assert something :)

The logging statement cannot be checked automatically but maybe the maxed out value and the actual landmark nodeIDs?